modules/mta.py

```diff
@@ -0,0 +1,84 @@
+# LtData
+# Copyright (C) 2015 Salvo "LtWorf" Tomaselli
+#
+# Relation is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# author Salvo "LtWorf" Tomaselli <tiposchi@tiscali.it>
+
+
+import json
+from time import strftime
+from os.path import normpath
+
+
+def openmbox(name):
+    mbox = normpath('%s/%s' % (config['files'], name))
```

Comment:
I'd rather have a subdirectory for that, so that you can't fuck up the other configuration files